web/app/integration/specs/daemonset/daemonset-table.js

```diff
@@ -0,0 +1,16 @@
+const assert = require('assert');
+    describe('table check', function() {
+    it('should check the page for proper table structure', () => {
+        browser.url('http://127.0.0.1:7777/daemonsets');
+        browser.waitUntil(() => {return $('table').isExisting();
+          }, 10000, 'expected table to exist');
+        http_headers=["Namespace", "Daemon Set","Meshed", "Success Rate", "RPS", "P50 Latency", 
```

Comment:
We use camel case for variables, so instead of `http_headers` we use `httpHeaders`

Reply:
Oh, I see. Will make the changes to my other test files as well. Thank you for the information.